### PR TITLE
Added Save/Load Highscore. <feature

### PR DESCRIPTION
**Pull request by dmalarev**
I added the save/load highscore function to the code.
If it gets merged, you need to put the text file in the gitignore.
Please, take a look and tell me what do you think.

### CODE REVIEW

File: src/main/FXML/MainScreen.fxml [hunk 1]
```diff
@@ -30,7 +30,7 @@
             <Font name="Consolas" size="33.0" />
          </font>
       </Text>
-      <Text fill="#ff1fc7" layoutX="63.0" layoutY="62.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Highest score achieved this session:" wrappingWidth="399.6708984375">
+      <Text fill="#ff1fc7" layoutX="63.0" layoutY="62.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Highest score:" wrappingWidth="399.6708984375">
```
Comment (ClintonCao) [inline src/main/FXML/MainScreen.fxml]: Can you also change the position of the text? Now we have a really big space between the text and the score. <review

**Comment by ClintonCao** [comment]
The rest of the code looks good to me. Nice job with the saving of the high score. <review

**Comment by ClintonCao** [comment]
Also don't forget to use the Octupull tags for the pull request!

**Comment by ClintonCao** [comment]
It looks much nicer now! < user-interface

**Comment by dmalarev** [comment]
I have edited the code in the Game class and now, according to inCode, it is no longer a Data Class.
Can anyone review it?

**Comment by ClintonCao** [comment]
Nice fix on the data class design flaw! I think we can merge it, if we all agree. <feature, organisation

**Comment by Michieldoesburg** [comment]
I agree we can merge it < review

**Pull request merged**